src/app/pages/sharing/nfs/nfs-form/nfs-form.component.html

```diff
@@ -122,6 +122,7 @@
           type="submit"
           color="primary"
           ixTest="save"
+          [ixMatButtonRoles]="[Role.SharingNfsWrite]"
```

Comment:
`ixRequireRoles`?

Reply:
This won't work on non mat-button elements. And we exclusively use mat-buttons on our forms so the naming should be reflective I think. Plus, it won't work on other non-button elements to don't want it to be mistakenly used on anything other than a mat-button.